src/index/api.rs

```diff
@@ -65,15 +63,15 @@ fn get_release_time_yanked_downloads(pkg: &MetadataPackage) -> Result<(time::Tim
             .and_then(|v| v.as_str())
             .ok_or_else(|| err_msg("Not a JSON object"))?;
 
-        if semver::Version::parse(version_num).unwrap().to_string() == pkg.version {
+        if Version::parse(version_num)?.to_string() == pkg.version {
             let release_time_raw = version
                 .get("created_at")
                 .and_then(|c| c.as_str())
                 .ok_or_else(|| err_msg("Not a JSON object"))?;
+
             release_time = Some(
-                time::strptime(release_time_raw, "%Y-%m-%dT%H:%M:%S")
-                    .unwrap()
-                    .to_timespec(),
+                DateTime::parse_from_str(release_time_raw, "%Y-%m-%dT%H:%M:%S%.f%:z")?
+                    .with_timezone(&Utc),
```

Comment:
Were past builds uploaded using UTC timezones? I'm not sure this is correct for existing builds.

Reply:
That might depend on what the server is set to (can you check that?). Hopefully it's UTC and everything just works out, otherwise a migration to update the times in the database could be necessary.

Reply:
The server is in UTC as far as I can tell.
```
$ date
Wed Jun  3 01:06:35 UTC 2020
```